source/core/TDCodecVia.cpp

```diff
@@ -1638,7 +1638,8 @@ void TDViaParser::ParseClump(VIClump* viClump, InstructionAllocator* cia)
     ClumpParseState state(viClump, cia, _pLog);
     SubString  token;
     SubString  instructionNameToken;
-    SubString  argExpressionTokens[ClumpParseState::kMaxArguments];
+    SubString  argExpressionTokenBacker[ClumpParseState::kMaxArguments];
+    ExpandableStaticArray<SubString> argExpressionTokens(argExpressionTokenBacker, ClumpParseState::kMaxArguments);
```

Comment:
I think it would be much cleaner for the ExpandableStaticArray to own the 'backer' static array as well. There's no advantage to having the caller pass it, and you have to duplicate the size and make sure it's consistent.

Reply:
Done